tests/src/integration/apigateway/manifest.yml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: book club
+  version: 1.0
+  license: Apache-2.0
+  apis: # new top-level key for defining groups of named APIs
```

Comment:
@mrutkows , just review the manifest file, no worry about other code, it's not finished yet.